.github/workflows/test_packages.yml

```diff
@@ -114,7 +114,7 @@ jobs:
           # - perception/object_detection_3d # passes, but disabled due to free() crash
           - perception/facial_expression_recognition
           - simulation/human_model_generation
-          - control/mobile_manipulation
+          # - control/mobile_manipulation
```

Comment:
Should we uncomment this test?
Or did you already try to run it and it is not working?

Reply:
No, I haven't tested.  I will uncomment it and check again.